build/scripts-3.11/import.py

```diff
@@ -0,0 +1,191 @@
+#!python
+
+import os
+import sys
+import ssl
+import glob
+import yaml
+import logging
+import argparse as ap
+import collections as col
+import dpimport.importer as importer
+from dpimport.database import Database
+from pymongo import DeleteMany, UpdateMany
+from pymongo.errors import BulkWriteError
+
+logger = logging.getLogger(__name__)
+
+def main():
+    parser = ap.ArgumentParser()
+    parser.add_argument('-c', '--config')
+    parser.add_argument('-d', '--dbname', default='dpdata')
+    parser.add_argument('-v', '--verbose', action='store_true')
+    parser.add_argument('expr')
+    args = parser.parse_args()
+
+    level = logging.INFO
+    if args.verbose:
+        level = logging.DEBUG
+    logging.basicConfig(level=level)
+
+    with open(os.path.expanduser(args.config), 'r') as fo:
+        config = yaml.load(fo, Loader=yaml.SafeLoader)
```

Comment:
This `config` isn't used anywhere, is it?

Reply:
yes it's used in the import.py file within the scripts directory